docs/api/browser-window.md

```diff
@@ -664,6 +664,13 @@ the player itself we would call this function with arguments of 16/9 and
 are within the content view--only that they exist. Just sum any extra width and
 height areas you have within the overall content view.
 
+#### `win.previewFile(pathname, filename)` _macOS_
+
+* `pathname` String - The absolute path to the file to preview with QuickLook. This is important as Quicklook uses the file name and file extension on the path to determine the content_type of the file to open.
+* `filename` String - The name of the file to display on QuickLook modal view. This is purely visual and does not affect the content_type of the file. 
```

Comment:
Just a drive by comment - I feel like `pathname` is a bit unusual. Maybe this should be `path` for consistency with other functions that take absolute paths, like https://github.com/electron/electron/blob/master/docs/api/native-image.md#nativeimagecreatefrompathpath?